UnitTests/expected/Storyboards-All-CustomName.swift.out

```diff
@@ -137,3 +158,31 @@ struct XCTStoryboardsSegue {
     case ShowPassword = "ShowPassword"
   }
 }
+
+extension XibCollectionViewCell { static var reuseIdentifier: String { return "XibCollectionViewCellReuseID" } }
+extension CustomCollectionViewCell { static var reuseIdentifier: String { return "CustomCollectionViewCellReuseID" } }
+extension CustomMessageCell { static var reuseIdentifier: String { return "CustomMessageCellReuseID" } }
+extension StandaloneCollectionViewCell { static var reuseIdentifier: String { return "StandaloneCollectionViewCellReuseID" } }
+extension StandaloneTableViewCell { static var reuseIdentifier: String { return "StandaloneTableViewCellReuseID" } }
+extension XibTableViewCell { static var reuseIdentifier: String { return "XibTableViewCellReuseID" } }
+
+struct XCTStoryboardsCell {
+  enum Anonymous: String, StoryboardCellType {
+    case ReuseCellID = "ReuseCellID"
+  }
+  enum CollectionViewCell: String, StoryboardCellType {
+    case XibCollectionViewCellReuseID = "XibCollectionViewCellReuseID"
```

Comment:
I think calling this `CollectionViewCell` a `StoryboardCellType` is the weird case as it would found in a XIB. Perhaps a `XibCellType` that simply extends `StoryboardCellType`?